documentation/md/style.md

```diff
@@ -168,7 +168,8 @@ Padding:
 
 A padding defines an addition to a node's dimension.  For example, `padding` adds to a node's outer (i.e. total) width and height.  This can be used to add spacing around the label of `width: label; height: label;` nodes, or it can be used to add spacing between a compound node parent and its children.
 
-* **`padding`** : The amount of padding around all sides of the node.
+* **`padding`** : The amount of padding around all sides of the node. Either percentage or pixel value can be specified. For example, both `50%` and `50px` are acceptable values. By default, percentage padding is calculated as a percentage of node width.
+* **`padding-relative-to`** : Determines how padding is calculated if and only if the percentage unit is used. If `width` is specified, the padding will be calculated as a percentage of the node width. If `height` is specified, the padding will be calculated as a percentage of the node height. If `whAverage` is specified, the padding will be calculated as a percentage of the average of the node height and width. If `whMin` is specified, the padding will be calculated as a percentage of the smallest number between width and height. If `whMax` is specified, the padding will be calculated as a percentage of the largest number between width and height.
```

Comment:
I think bullet form would be easier to read as opposed to a sequence of if statements.
e.g

* ```whAverage```:  padding will be the average of the node height and width
* ```whMin```:  ....

It will be easier for people to understand what these properties actually do.